ti/api/query.js

```diff
@@ -1,86 +1,91 @@
-import {fireEvent} from "./event";
+import { fireEvent } from './event'
+import Handlebars from 'handlebars'
+import Config from '../store/config'
 
-let cache = {};
-let cacheEnabled = false;
-let cacheTimeout = 500;
+let cache = {}
+let cacheEnabled = false
+let cacheTimeout = 500
 
-function cacheQuery(query, result) {
+function cacheQuery (query, result) {
     cache[query] = {
         result: result,
         time: Date.now()
-    };
+    }
 }
 
-function clearCache() {
+function clearCache () {
     cache = {}
 }
 
-function hasCached(query) {
+function hasCached (query) {
     return cacheEnabled
         && !query.toLowerCase().includes('delete')
         && !query.toLowerCase().includes('insert')
         && cache.hasOwnProperty(query)
-        && cache[query].time + cacheTimeout > Date.now();
+        && cache[query].time + cacheTimeout > Date.now()
 }
 
-function query(query, db = 'pitcher') {
+function query (query, db = 'pitcher') {
```

Comment:
Standard is no space after the function name

Reply:
somehow newest intellij autoformater thinks this is a greate idea... disabled again